### PR TITLE
Query paramater

### PR DESCRIPTION
**Pull request by kusumashivanna**
screenshot-localhost-3000-2019 08 22-17-56-20

### CODE REVIEW

File: src/container/NewsFeedContainer.js [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-import Newsfeed from "../component/Newsfeed.js";
-import { GetNews } from "../action/Newsfeedaction.js";
+import NewsFeed from "../component/NewsFeed.js";
+import { getNews } from "../action/NewsFeedAction.js";
 
 const mapDispatchToProps = dispatch => {
   return {
-    GetNew: () => {
-      dispatch(GetNews());
+    getNew: () => {
```
Comment (subash-v) [inline src/container/NewsFeedContainer.js]: give a proper name for the function

**Review by subash-v — CHANGES_REQUESTED** [review]

File: src/container/NewsFeedContainer.js (hunk 1 above)
Comment (banupriya-SR-xelp) [reply src/container/NewsFeedContainer.js]: change the function name 
as "getNewsFeed()"

File: src/container/NewsFeedContainer.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+import NewsFeed from "../component/NewsFeed.js";
+import { getNews } from "../action/NewsFeedAction.js";
+
+const mapDispatchToProps = dispatch => {
+  return {
+    getNew: () => {
+      dispatch(getNews());
```
Comment (banupriya-SR-xelp) [inline src/container/NewsFeedContainer.js]: change function name as dispatch(getNewsFeed())

File: src/container/NewsFeedContainer.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    }
+  };
+};
+const mapStateToProps = state => {
+  return {
+    userData: state.newsData
```
Comment (banupriya-SR-xelp) [inline src/container/NewsFeedContainer.js]: change userData has newsFeed

**Review by banupriya-SR-xelp — CHANGES_REQUESTED** [review]

**Review by banupriya-SR-xelp — APPROVED** [review]

**Pull request merged**